administration/src/bp-modules/regions/freinet/FreinetSettingsCard.tsx

```diff
@@ -0,0 +1,71 @@
+import { Button, Checkbox, H2 } from '@blueprintjs/core'
+import React, { ReactElement, useState } from 'react'
+import { Trans, useTranslation } from 'react-i18next'
+import styled from 'styled-components'
+
+import { FreinetAgency } from '../../../generated/graphql'
+import SettingsCard from '../../user-settings/SettingsCard'
+
+const Headline = styled(H2)`
+  margin-bottom: 16px;
+`
+
+const ButtonContainer = styled.div`
```

Comment:
Even the settings card could be refactored to reuse code. I think we can keep it for now, since we probably gonna redesign this with mui components at some point